src/services/dataCenterRegistry/index.ts

```diff
@@ -55,7 +55,7 @@ async function fetchDataCenter(code: string): Promise<DataCenter | undefined> {
   })
     .then(async (response: Response) => (await response.json()) as DataCenter)
     .catch(async (response: Response) => {
-      const responseJson = await response.json();
+      const responseJson: any = await response.json();
```

Comment:
since there's a Response type, is there any chance there's a Response['json']-esque type instead of `any` ?

Reply:
just an object with a string message property. we're not returning responseJson, which would be Promise return type

Reply:
updated